src/main.ts

```diff
@@ -93,10 +93,10 @@ export class AutoRelease {
     this.semVerLabels = defaultLabels;
 
     if (rawConfig.labels) {
-      this.semVerLabels = {
+      this.semVerLabels = new Map<VersionLabel, string>([
         ...defaultLabels,
-        ...rawConfig.labels
-      };
+        ...(Object.entries(rawConfig.labels) as any)
```

Comment:
@orta I couldn't figure out what to do w/ the types here. Any suggestions? Technically the `Object.entries` here should be returning something like `Array<[VersionLabel, string]>`. I tried several things, but I eventually just gave up.

Reply:
https://github.com/intuit/auto-release/blob/master/src/github-release.ts#L102

You could do `[VersionLabel, string]` but it would have the same effect I think. Can you try that out? We don't have many anys in the codebase

Reply:
Yep, I got it. Thanks for the tip.